runtime/gc_structs/ClassIteratorClassSlots.cpp

```diff
@@ -77,7 +77,14 @@ GC_ClassIteratorClassSlots::nextSlot()
 			return classPtr;
 		}
 		_state += 1;
-		
+
+	case classiteratorclassslots_state_flattened_class_cache_slots:
+		classPtr = _classFCCSlotIterator.nextSlot();
+		if (NULL != classPtr) {
+			return classPtr;
+		}
+		_state += 1;
+
```

Comment:
Would you please add formal state `classiteratorclassslots_state_end:` here (just before `default:`)?